org.eclipse.gef.mvc.fx/src/org/eclipse/gef/mvc/fx/gestures/DragDropGesture.java

```diff
@@ -0,0 +1,276 @@
+/*******************************************************************************
+ * Copyright (c) 2018 KDM Analytics Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Kyle Girard (KDM Analytics Inc.) - initial API and implementation
+ *
+ *******************************************************************************/
+
+package com.kdma.brm.riskmap.gestures;
```

Comment:
The package name `com.kdma.brm.riskmap.gestures` seems to be weird. Please update the PR to use proper GEF package names.